docs/dashboard/dashboard.rst

```diff
@@ -0,0 +1,275 @@
+
+.. figure:: /dashboard/images/img_7.png
+   :width: 100%
+   :align: center
+   :alt: Version buttons
+
+They are, in order from left to right:
+
+- Perform dry run: the orchestrator will go through all resources in the model and compare their current state to their desired state. This can be useful to double-check what effect the deployment of a version might have on your current environment.
+- Dry run report: will take you to the report of the last performed dry run, without performing a new one.
+- Release version: If ``auto-deploy`` in the ``environment settings`` is set to ``False``, this button can be used to deploy the model, otherwise this button will be grayed out.
+- Remove version: removes the selected version from the inmanta environment.
+
+Finally, clicking the version number will take us to the overview of that particular version.
+It gives the same options as the version overview does and it displays a list of all resources and their current state.
+
+.. figure:: /dashboard/images/img_8.png
+   :width: 100%
+   :align: center
+   :alt: Resource overview
+
+Using the filters we can filter for resources by type, by agent used to deploy the resource, value and, maybe most importantly, by deploy state.
+This display is continuously updated, both during deploys and after, when the orchestrator goes through all resources to make sure they remain in the desired state.
+
+Taking a closer look at the a specific resource, there are 2 important buttons, the ``Dependency`` button and a ``magnifying glass``.
+The ``Dependency`` button is only available if a resource depends on other resources.
+When pressed it will add lines to the table displaying the each dependency and it's current state:
+
+.. figure:: /dashboard/images/img_9.png
+   :width: 100%
+   :align: center
+   :alt: Resource dependencies
+
+   Resource dependencies
+
+The `magnifying glass` icon will take us to an in depth overview of the resource.
+This will show a complete breakdown of the resource's desired state at the top and.
+
+The desired state breakdown allows for easy inspection of the impact the resource will have.
+For example, the resource in the image below will deploy a file with path `/etc/my.cnf` and file permission `644`.
+We can even inspect the file's content.
+
+.. figure:: /dashboard/images/img_10.png
+   :width: 100%
+   :align: center
+   :alt: Resource view
+
+   Resource desired state view
+
+Below is the action log, shows a log of actions taken on the given resource.
+This varies from dry-runs to deploys.
+This log will typically start filling up with deploys due to the orchestrator enforcing the desired state.
+Again we can further inspect an action by pressing the drop down arrow.
+
+.. figure:: /dashboard/images/img_11.png
+   :width: 100%
+   :align: center
+   :alt: Resource action
+
+   Resource action view
+
+Each of these logs can then be further analyzed by pressing the ``magnifying glass``.
```

Comment:
Is there a way to include the icon in the documentation?  Not that it is an absolute requirement, but I think it would look cool.

Reply:
hmm... maybe with unicode? @arnaudsjs  you have an idea about this?

Reply:
On the dashboard, this is generated using bootstrap `glyphicon glyphicon-zoom-in` (`<span class="glyphicon glyphicon-zoom-in"></span>`). So if you can render html in rst files, you can include it. I am not sure how easy that is.

Reply:
you can with the `raw` directive, but it would be a seperate element and thus not in the sentence :/

Reply:
you can with the raw directive, but it would be a seperate element and thus not in the sentence :/

Reply:
Forget it then, it is not that important